src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/customer.html.twig

```diff
@@ -44,9 +44,7 @@
 
               <strong class="text-muted ml-2">#{{ orderForViewing.customer.id }}</strong>
             </h2>
-            {% if orderForViewing.customer.isGuest %}
-              <strong class="text-muted">Guest</strong>
-            {% endif %}
```

Comment:
I think we should keep this, if a customer is in multiple groups, the information about him being a guest is easier to get here.

Reply:
@atomiix It's gonna look the same. If you do not show groups for guests, you loose possible information, if some merchant assigns other groups to guests.

Reply:
Why would it be weird to display both? In one case, "Guest" is a group the customer is in, in the 2nd case, it's more of a "status". And as you can define other groups as guest, it may be important to remind that a customer is considered a guest by PrestaShop even though he's not in any group named "Guest".

Reply:
It's gonna look like this. 
![Výstřižek](https://user-images.githubusercontent.com/6097524/146749592-5a3aece2-1f3b-4b0d-80c0-c2f652f06e47.JPG)

The design was already validated and specified by Scott, please don't change it...